src/files/naxsi_include_module.conf

```diff
@@ -1 +0,0 @@
-include       /etc/nginx/naxsi_core.rules;
```

Comment:
This is now being included in-line within the http block which I think makes sense and removes another extra file.